dev/io.openliberty.microprofile.openapi.ui.internal_fat/fat/src/io/openliberty/microprofile/openapi/ui/internal/fat/tests/UIOauthTest.java

```diff
@@ -218,11 +228,14 @@ public void OAuthTest(String uiPath) {
         allowOnceBtn.click();
 
         // should now have a single tab which is the original openapi page
-        new WebDriverWait(driver, Duration.ofSeconds(3)).until(driver -> driver.getWindowHandles().size() == 1);
+        new WebDriverWait(driver, Duration.ofSeconds(3)).until(d -> driver.getWindowHandles().size() == 1);
```

Comment:
I don't _think_ this changes anything. Was this change (and the one on line 210) just testing to see if it made a difference?

Reply:
This is now in line with the Selenium documentation for waiting, this was to see if this would improve some of the other behaviors we have seen on the build systems. though we typically saw the second one have issues, not the first.

Reply:
Huh, fair enough